docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -20,14 +20,15 @@ session.set_providers(['OpenVINOExecutionProvider'], [{Key1 : Value1, Key2 : Val
 *Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization*
 
 ### C/C++ API
-All the options (key-value pairs) need to be concantenated into a string as shown below and passed to OrtSessionOptionsAppendExecutionProviderEx_OpenVINO() API as shown below:-
+All the options shown below are passed to SessionOptionsAppendExecutionProvider_OpenVINO() API and populated in the struct OrtOpenVINOProviderOptions in an example shown below:-
 
 ```
-std::string settings_str;
-settings_str.append("Key1|Value1\n");
-settings_str.append("Key2|Value2\n");
-settings_str.append("Key3|Value3\n");
-Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProviderEx_OpenVINO(sf, settings_str.c_str()));
+OrtOpenVINOProviderOptions options;
+options.device_type = "CPU_FP32";
+options.enable_vpu_fast_compile = 0;
+options.device_id = "";
+options.num_of_threads = 8;
+SessionOptionsAppendExecutionProvider_OpenVINO(session_options, &options);
 ```
 
 ### Available configuration options
```

Comment:
In the available configuration options, should we be also updating the data type of **value_type** field?
Because, Ryan has made some changes.

typedef struct OrtOpenVINOProviderOptions {
#ifdef __cplusplus
  OrtOpenVINOProviderOptions() : device_type{}, enable_vpu_fast_compile{}, device_id{}, num_of_threads{} {}
#endif
  **const char*** device_type;                // CPU_FP32, GPU_FP32, GPU_FP16, MYRIAD_FP16, VAD-M_FP16 or VAD-F_FP32

  **unsigned char** enable_vpu_fast_compile;  // 0 = false, nonzero = true
  **const char*** device_id;

  size_t num_of_threads;  // 0 uses default number of threads


} OrtOpenVINOProviderOptions;

so it should be,
for device_type,                                      value_type:  **const char**
for enable_vpu_fast_compile,                 value_type:    **unsigned char**
for device_id,                                          value_type:  **const char**

Reply:
i think we can just let it be, it wont make much of a difference